panic_handlers/small_panic/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libtock_small_panic"
+authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
+version = "0.1.0"
+description = """Default panic handler for libtock. Calls low-level debug on panic and yields."""
```

Comment:
Please update the description. This isn't the default panic handler for `libtock` (the debug handler will be), and it exits rather than yields.

Reply:
Done